walkthroughs/1-ionic-showcase/walkthrough.json

```diff
@@ -0,0 +1,11 @@
+{
+  "dependencies": {
+    "repos": [],
+    "managedServices": [
+      {
+        "name": "amq-online-standard"
```

Comment:
I'm not 100% sure how this works... But we should be using a `brokered` address space not a standard one :)